test/test.js

```diff
@@ -57,9 +57,22 @@ test('fly-rollup', t => {
         t.true(res.indexOf('# sourceMappingURL=data:application/json;base64,{"version":3,') > -1, 'appends `sourceMappingURL` content');
```

Comment:
@frantic1048 I'm not fully understanding.

As-is — i.e. `new Buffer(JSON.stringify(res.map).toString('base64'));` — the result isn't actually being base64 encoded (thus why the tests pass with `{"version":3`).

Should this be fixed and `{"version":3` removed from the test?